fk/secretreceive.go

```diff
@@ -202,13 +230,77 @@ func countDigits(i int) (count int) {
 	return len(iString)
 }
 
+func getFileNameToWrite(filename string) (string, error) {
+	userDir, err := homedir.Dir()
+	if err != nil {
+		return "", err
+	}
+	downloadsDir := userDir + "/Downloads"
```

Comment:
This should be `filepath.Join(..)` (it's generally unsafe to concatenate directory separators as strings, and not cross-platform, in case we ever supported Windows)

Reply:
Whoops! Yep, I came across this concern in writing this, but missed this one! Thanks!